HermaFx.DataAnnotations/MinAttribute.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+
+namespace HermaFx.DataAnnotations
+{
+	/// <summary>
+	/// Can be used with parameter target..
+	/// </summary>
+	[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
+	public class MinAttribute : ValidationAttribute
+	{
+		#region Fields
+
+		private const string ERROR_MESSAGE = "The field {0} should be greater or equal than {1}";
```

Comment:
This should use the 'GetErrorMessage' pattern (as in MinElementsAttribute, etc.) so we can override the message for different locales.

Reply:
Let me check